libs/eo/landing-page/shell/src/lib/eo-landing-page-header.component.ts

```diff
@@ -57,12 +57,14 @@ const selector = 'eo-landing-page-header';
           background: var(--watt-color-primary);
           color: var(--watt-color-primary-contrast);
 
-          // This is equivalent to a normal size watt button
+          // This is equivalent to a normal size watt button width
           min-width: 6.25rem;
-          height: 2.75rem;
+          height: calc(5 * var(--watt-space-s)); // 40px
+          padding: var(--watt-space-s) var(--watt-space-m) var(--watt-space-s)
+            var(--watt-space-m); // 8px 16px 8px 16px
```

Comment:
question: What does this match? Add a comment referring to the Inset Squish or whatever style its matching.
issue: Remove the comment with a value that can get out of sync.